src/scripts/containers/ProfileMyVideosContainer.js

```diff
@@ -0,0 +1,15 @@
+/* @flow */
+
+import { connect } from 'react-redux'
+
+import ProfileMyVideos from 'components/ProfileMyVideos'
+import { getUploaderVideos } from 'selectors/UploaderSelectors'
+import type { RootState } from 'types/ApplicationTypes'
+
+const mapStateToProps = (state: RootState) => ({
+  videos: getUploaderVideos(state)
+})
+
+const mapDispatchToProps = () => ({})
+
+export default connect(mapStateToProps, mapDispatchToProps)(ProfileMyVideos)
```

Comment:
FYI if `mapDispatchToProps` does nothing you can simply just pass nothing in as the second argument

Reply:
Removing

Reply:
@bent0b0x if I remove the second arg the container fails on `run quality`

```
Error: src/scripts/containers/ProfileMyVideosContainer.js:13
 13: export default connect(mapStateToProps)(ProfileMyVideos)
                    ^^^^^^^^^^^^^^^^^^^^^^^^ call of `connect`. Could not decide which case to select
 13: export default connect(mapStateToProps)(ProfileMyVideos)
                    ^^^^^^^ intersection type
  Case 3 may work:
                                 v--------------
   98:   declare function connect<S, A, OP, SP>(
   99:     mapStateToProps: MapStateToProps<S, OP, SP>,
  100:     mapDispatchToProps: Null,
  ...:
  103:   ): Connector<OP, $Supertype<SP & { dispatch: Dispatch<A> } & OP>>;
         ----------------------------------------------------------------^ function type. See lib: flow-typed/npm/react-redux_v5.x.x.js:98
  But if it doesn't, case 5 looks promising too:
                                 v------------------
  112:   declare function connect<S, A, OP, SP, DP>(
  113:     mapStateToProps: MapStateToProps<S, OP, SP>,
  114:     mapDispatchToProps: MapDispatchToProps<A, OP, DP>,
  ...:
  117:   ): Connector<OP, $Supertype<SP & DP & OP>>;
         -----------------------------------------^ function type. See lib: flow-typed/npm/react-redux_v5.x.x.js:112
  Please provide additional annotation(s) to determine whether case 3 works (or consider merging it with case 5):
                                                      v
    9: const mapStateToProps = (state: RootState) => ({
   10:   videos: getUploaderVideos(state)
   11: })
       ^ return


Found 1 error
```

Reply:
Done!